package exceptions;

public class ExceptionDemo {
	
	int a = 4;
	int b = 7;
	int c = 0;

	
	public static void main(String[] args) {
		// TODO Auto-generated method stub
		try {
			int k=b/c;
		} catch (Exception e) {
			// TODO: handle exception
			System.out.println(e);
		}
	}

}
